x-pack/plugins/elastic_assistant/server/routes/helpers.ts

```diff
@@ -681,7 +681,8 @@ export const getIsKnowledgeBaseEnabled = async (
         esqlExists =
           (
             await kbDataClient.getKnowledgeBaseDocumentEntries({
-              query: ESQL_DOCS_LOADED_QUERY,
+              query: '',
+              kbResource: ESQL_RESOURCE,
               required: true,
             })
```

Comment:
Same ⚠️ here as noted in https://github.com/elastic/kibana/pull/184885#discussion_r1783443591